features/step_definitions/account_steps.rb

```diff
@@ -1,119 +1,118 @@
+When "account for {string} has been approved" do |user_email|
+  User.find_by!(email: user_email).account.approve!
 end
 
-Then /^I should see the page to change account details$/ do
+Then "I should see the page to change account details" do
   response.should have_tag('form') do
     with_tag 'input#account_org_name'
   end
 end
 
 # TODO: Move these over to a separate steps file.
-Then /^I should be able to edit the value of the customers type field$/ do
+Then "I should be able to edit the value of the customers type field" do
   response.should have_xpath("//input[@name='account[profile_attributes][customers_type][]']")
 end
 
-Then /^I should not be able to edit the value of the customers type field$/ do
+Then "I should not be able to edit the value of the customers type field" do
   response.should_not have_xpath("//input[@name='account[profile_attributes][customers_type][]']")
 end
 
-Then /^I should see the value of the customers type field is "([^\"]*)"$/ do |value|
+Then "I should see the value of the customers type field is {string}" do |value|
   response.should have_xpath("//input[@id='account_profile_attributes_customers_type_#{value.downcase}' and @checked='checked']")
 end
 
-Then /^(account "[^\"]*") should be (pending|approved|rejected)$/ do |account, state|
+Then "{account} should be {state}" do |account, state|
   assert_equal state, account.state
 end
 
-Then /^I should not be able to cancel account$/ do
-  assert has_no_css? 'a', :text => "Cancel account"
+Then "I should not be able to cancel account" do
+  assert has_no_css? 'a', text: "Cancel account"
 end
 
-Then /^I should see the notice that I will receive the invoice$/ do
+Then "I should see the notice that I will receive the invoice" do
   assert has_content? "Also note that you have pending invoices. You will receive an email with the exact amount you will be charged on the end of the month"
 end
 
-Then /^I should see the account details:$/ do |table|
+Then "I should see the account details:" do |table|
   table.diff! extract_table('#account-overview', 'tr', 'th,td')
 end
 
-Then /^(provider "[^"]*") time zone should be "([^"]*)"$/ do |provider, time_zone|
+Then "{provider} time zone should be {string}" do |provider, time_zone|
   provider.timezone.should == time_zone
 end
 
-Then /^the provider time zone is "([^"]*)"$/ do |time_zone|
-  @provider.update_column(:timezone, time_zone)
+Then "the provider time zone is {string}" do |time_zone|
+  @provider.update!(timezone: time_zone)
 end
 
-Then /^(account "[^"]*") should be (provider|buyer|master)$/ do |account,type|
+Then /^(account "([^"]*)") should be (provider|buyer|master)$/ do |account, type|
   assert account.send("#{type}?"), "Account '#{account.org_name}' is not a #{type}"
 end
 
-Given /^((?:account|buyer|provider) "[^"]*") has only one admin "([^"]*)"$/ do |account, username|
-  to_be_admin = account.users.find_by_username!(username)
+Given "{buyer} has only one admin {string}" do |buyer, username|
```

Comment:
This step name doesn't click to me. Is it more like `{buyer} has only user {string} as admin`? Or do I misinterpret?